docs/web/index.md

```diff
@@ -80,17 +79,38 @@ provided by the Open Policy Agent to decide about denying anonymous requests.**
 Spring Security is disabled for the Management/Admin Port (default: 8081). Be aware that these port
 should not be accessible out of the deployment context.
 
+This security implementation lacks some features compared to [sda-dropwizard-commons](https://github.com/SDA-SE/sda-dropwizard-commons/tree/master/sda-commons-server-auth):
+- No configuration of static local public keys to verify the token signature. 
+- No configuration of JWKS URIs to verify the token signature. 
+- The IDP must provide an `iss` claim that matches the base URI for discovery. 
+- Leeway is not configurable yet. 
+- The client that loads the JWKS is not configurable yet.
```

Comment:
@JoergSiebahn: Is this list still accurate? I remember that @saadmar added some more features but can't find the commit.